tests/data/test_daily_bars.py

```diff
@@ -363,14 +382,17 @@ def test_write_ohlcv_content(self):
             multiplier = 1 if column == 'volume' else 1000
             for asset_id in self.assets:
                 for date in self.dates_for_asset(asset_id):
-                    self.assertEqual(
-                        expected_bar_value(
+                    if asset_id in HOLES and date in HOLES[asset_id]:
+                        expected = 0
+                    else:
+                        expected = expected_bar_value(
```

Comment:
I noted this below as well: this might be clearer if we passed HOLES all the way down to `expected_bar_value`, since that's what the 2d version is calling anyway.